src/main/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask.java

```diff
@@ -152,6 +155,14 @@ public void setMaxWarnings(int maxWarnings) {
     }
 
     /**
+     * Adds a source path.
+     * @param sourcePath the source path to add.
+     */
+    public void addSourcepath(Path sourcePath) {
```

Comment:
variable is named `sourcePath` so why is method named `Sourcepath`? It should be `SourcePath` to have similar casing.